Edit pass and date format update on Simple Exchange (#283)
Agoric/documentation: main/zoe/guide/contracts/simple-exchange.md

<Zoe-Version/>

> ##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 12-SEP-2020)

Reviewer: for dates, I actually prefer 2020-09-12. (You're right that I used the US-only format. shame on me.)  It seems to be an ISO standard, and it seems to be uniquely recognizable both US and internationally.

Author: Makes sense. Used 12-SEP-2020 because that's what's currently on the site wide variable (changed from our previous 9/12/2020 style, so it wasn't just you). I can argue for why day-month is usually the most relevant info first in these cases, but ISO standard beats that.

##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)

The "simple exchange" is a very basic, un-optimized exchange. It
has an order book for one asset, priced in a second asset. The order
book is naively kept as an array that the contract iterates over for matches
when a new order arrives.

## Contract API

SimpleExchange is an exchange with a simple matching algorithm. This lets
an unlimited number of parties create new orders or accept existing
orders. A notifier lets callers find the current list of orders.

The SimpleExchange keywords are `Asset` and `Price`. The contract treats
the two keywords symmetrically. New offers are created and existing offers
accepted in either direction.

```js
{ give: { Asset: simoleans(5) }, want: { Price: quatloos(3) } }
{ give: { Price: quatloos(8) }, want: { Asset: simoleans(3) } }
```

> The `want` term is an exact amount to exchang, while the

Reviewer: ```suggestion
The `want` term is an exact amount to exchange, while the
```

`give` term is a limit that may be improved on. This simple exchange does not
partially fill orders.

> The `publicFacet` is returned when the contract is started.

Reviewer: Do we need 'the' with the highlighted term?
```suggestion
`publicFacet` is returned when the contract is started.
```

Author: I added the "the" to emphasize the "what it is" that this is in addition to "name of object"  here. By emphasizing that "public facet" is a thing and not just a name, it does a small bit of encouragment to look it up and find out what a public facet is if one doesn't know.

## Payout

> When a match is made, the payout promise in a user's seat

Reviewer: ```suggestion
When a match is made, the payout promise from a user's seat
```

const bobSimGainAmount = await E(bobSimPurse).deposit(bobPricePayout);
```

> Alice gets her payouts the same way. (The choice of `getPayouts()` vs
> `getPayout(keyword)` is purely for convenience).

Reviewer: This didn't sound smooth when I wrote it, and now that I'm seeing it again, I want to improve it.
```suggestion
Alice gets her payouts the same way. (The choice between `getPayouts()` and
`getPayout(keyword)` is based on which is more convenient in each circumstance).
```
